specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Override_TargetFrameworks_with_TargetFrameworks.cs

```diff
@@ -4,8 +4,7 @@ public class Reports
 {
     [Test]
     [Ignore("Buildalizer does not output any artifacts, hence nothing is analyzed.")]
-    public void TFM_overriding_TFMs()
-        => new OverrideTargetFrameworksWithTargetFrameworks()
+    public void TFM_overriding_TFMs() => new OverrideTargetFrameworksWithTargetFrameworks()
```

Comment:
The wording in the code makes it seem like `TFM` is just shorthand for the `TargetFramework` node and `TFMs` is shorthand for the `TargetFrameworks` node. While TFM (Target Framework Moniker) just means the string indicating the framework. i.e. `net7.0`, or `net8.0`.

Reply:
Suggestion for better naming?